pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Transactions.java

```diff
@@ -71,4 +73,21 @@
      */
     CompletableFuture<TransactionMetadata> getTransactionMetadata(TxnID txnID);
 
+    /**
+     * Get transaction buffer stats.
+     *
+     * @param topic the topic of getting transaction buffer stats
+     * @return the future stats of transaction buffer in topic.
+     */
+    CompletableFuture<TransactionBufferStats> getTransactionBufferStats(String topic);
+
+    /**
+     * Get transaction pending ack stats.
+     *
+     * @param topic the topic of this transaction pending ack stats
+     * @param subName the topic of this transaction pending ack stats
```

Comment:
```suggestion
     * @param subName the subName of this transaction pending ack stats
```
is `subName` accurate? or `subscription name`?